package.json

```
@@ -77,13 +77,12 @@
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.6.0",
+    "@ember/test-helpers": "^2.9.2",
```

Let's actually target the latest: 2.9.3 (which has been out for ~5 months or something)!

```suggestion
    "@ember/test-helpers": "^2.9.3",
```

(It won't change the output; Yarn correctly resolved it to that already, but it's best to be explicit.)

---

Gotcha, yeah sure.